### PR TITLE
Show branding in stickies

### PR DESCRIPTION
**Pull request by callumlocke**
looks like this:

image

### CODE REVIEW

**Comment by lc512k** [comment]
:+1:

File: components/stickies/sticky-item/_main.scss [hunk 1]
```diff
@@ -21,6 +21,16 @@
 	margin: 7px 0;
 }
 
+.sticky-item__branding {
+	> a {
```
Comment (andygnewman) [inline components/stickies/sticky-item/_main.scss]: the nLinksTopic mixin is our standard way of doing tag links.

Comment (callumlocke) [inline components/stickies/sticky-item/_main.scss]: yeah I was going to use that, but apparently we've been given the ok from next design team to try out this design in the brexit stickies:

image

(haven't actually done the big quote marks yet, but that will come)

**Comment by callumlocke** [comment]
thanks!

File: components/stickies/sticky-item/_main.scss (hunk 1 above)
Comment (andygnewman) [inline components/stickies/sticky-item/_main.scss]: We already use this type of styling in the article app (eg. https://next.ft.com/content/a97efd1e-d34c-11e5-829b-8564e7528e54) and it's on the home page (usually found in the opinion section).

**Pull request merged**